src/careamics/lightning/lightning_module.py

```diff
@@ -692,6 +713,15 @@ def reduce_running_psnr(self) -> float | None:
             psnr = np.mean(psnr_arr)
         return psnr
 
+    def _get_device(self):
```

Comment:
Can we put this in `utils`? It feels we have duplicated this code in several places.